test/addons/openssl-binding/test.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
 
 const common = require('../../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  process.exit(0);
```

Comment:
In this case I don't think we can just return as that would cause the tests that require `tls-connect` should not be able to proceed if there is no crypto. There might be other/better ways to deal with this but perhaps that should be done as a separate PR in that case.